app/modules/ads/setup.js

```diff
@@ -179,6 +179,12 @@ export const adsSetup = {
     context.set('services.moatYi.enabled', instantConfig.isGeoEnabled('wgAdDriverMoatYieldIntelligenceCountries'));
     context.set('services.nielsen.enabled', instantConfig.isGeoEnabled('wgAdDriverNielsenCountries'));
 
+    if (instantConfig.get('icTaxonomyComicsTag')) {
+      context.set('services.taxonomy.comics.enabled', instantConfig.get('icTaxonomyComicsTag'));
```

Comment:
From my understanding we later check only if this flag is true so maybe just set it to true? If instantConfig.get('icTaxonomyComicsTag') is not true we won't get here anyway.